users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -319,8 +531,9 @@ def run_scf_audio_perturbation():
         {"tempo": {"prob": 0.6, "minimum": 0.83, "maximum": 1.17}},
         {"tempo": {"prob": 0.6, "minimum": 0.9, "maximum": 1.1}},
         {"tempo": {"prob": 0.6, "minimum": 0.8, "maximum": 1.2}},
-        {"preemphasis": {"prob": 0.9, "minimum": 0.9, "maximum": 1.0, "default": 0.97}},
-        {"preemphasis": {"prob": 1.0, "minimum": 1.0, "maximum": 1.0, "default": 0.97}},
+        {"preemphasis": {"prob": 0.3, "minimum": 0.94, "maximum": 1.0, "default": 0.97}},
+        {"preemphasis": {"prob": 0.7, "minimum": 0.94, "maximum": 1.0, "default": 0.97}},
+        {"preemphasis": {"prob": 1.0, "minimum": 0.94, "maximum": 1.0, "default": 0.97}},
```

Comment:
I am not sure if preemphasis with an alpha > 1 makes sense and since the default is 0.97 the range is not really changeable

Reply:
It would still be possible to lower the minimum. Certainly, this would have the disadvantage of a default that is not the mean value of the perturbations. But I think you could give it a try with one experiment at least.

Reply:
I will add it to the experiments